src/shutdown/umount.c

```diff
@@ -155,6 +160,22 @@ int mount_points_list_get(const char *mountinfo, MountPoint **head) {
                 if (!m)
                         return log_oom();
 
+                iter_children = mnt_new_iter(MNT_ITER_FORWARD);
+                if (!iter_children)
+                        return log_oom();
+
+                struct libmnt_fs *child;
+                r = mnt_table_next_child_fs(table, iter_children, fs, &child);
+                if (r < 0) {
+                        return log_error_errno(r, "Failed to get children mounts for %s from %s: %m", path, mountinfo ?: "/proc/self/mountinfo");
+                } else if (r == 1) {
```

Comment:
no {} around single-line if blocks, as per coding style.

also: given the first `if` branch here exits the function you can drop the `else` without change of effect, it's redundant

Reply:
Done